apex/python/log.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import time
+
+class message:
+    def __init__(self):
```

Comment:
I assume this is a generic logger class right ? Maybe have an optional file name here for the Logger to reflect the object it is logging